src/developer/web-api/tracker.md

```diff
@@ -1810,8 +1810,8 @@ Table: Assigned user modes
 | CURRENT | Includes events assigned to the current logged in user. |
 | PROVIDED | Includes events assigned to the user provided in the request. |
 | NONE | Includes unassigned events only. |
-| ANY | Includes all assigned events, doesn't matter who are they assigned to as long as they assigned to someone. |
-| ALL | Includes all events irrespective of whether a user is assigned. If no mode is specified, this one is used by default.|
+| ANY | Includes all assigned events, regardless of who they are assigned to, as long as they are assigned to someone. |
+| ALL | Includes all events irrespective of whether a user is assigned. This is the default mode. |
```

Comment:
Can you please add to the request param table that `ALL` is the default mode.

Reply:
It's there already, you mean [here](https://github.com/dhis2/dhis2-docs/pull/1473/files#diff-4b5dabc4706dd8ae8c4c71cc43b6c8d08fa520c861a63cae3908ac407f201fffR1794) right?